Rakefile

```diff
@@ -38,4 +40,25 @@ namespace :beaker do
     args.with_defaults(:test_files => 'acceptance/performance/')
     run_beaker(args[:test_files])
   end
+
+  desc "Run beaker based acceptance tests, leaving VMs in place for later runs"
+  task :first_run, [:test_files] do |t, args|
+    args.with_defaults(:test_files => 'acceptance/tests/')
+    ENV["BEAKER_PRESERVE_HOSTS"] = "always"
+    run_beaker(args[:test_files])
+  end
+
+  desc "Re-run beaker based acceptance tests, using previously provisioned VMs"
+  task :rerun, [:test_files] do |t, args|
+    args.with_defaults(:test_files => 'acceptance/tests/')
+    ENV["PUPPETDB_SKIP_INSTALL"] = "true"
+    ENV["BEAKER_NO_PROVISION"] = "true"
```

Comment:
In any case, the distinction here is because they're controlling different parts of the system. I think it's useful to directly expose what beaker is doing, and the PUPPETDB stuff messes with our own code.